index.html

```diff
@@ -6,7 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="styles.css">
   <script src="data.js"></script>
-  <script src="index.js"></script>
+  <script src="index.js" defer></script>
```

Comment:
Great! You can also move your script tag to just above the `</body>` tag to handle this, but we'll ALSO learn a new tool that will help us with this soon.